src/bot.coffee

```diff
@@ -206,9 +209,20 @@ class SlackBot extends Adapter
   error: (error) =>
     @robot.logger.error "Slack RTM error: #{JSON.stringify error}"
     # Assume that scripts can handle slowing themselves down, all other errors are bubbled up through Hubot
-    # NOTE: should rate limit errors also bubble up?
     if error.code isnt -1
       @robot.emit "error", error
+    else
```

Comment:
I can't find any official documentation of the rate-limiting code, but this test indicates that code -1 is reliably rate limiting: https://github.com/slackapi/hubot-slack/blob/master/test/bot.coffee#L168-L171